src/elements/checkbox/checkbox-group/checkbox-group.stories.ts

```diff
@@ -112,9 +112,7 @@ const ErrorMessageTemplate = ({
 }: Args): TemplateResult => html`
   <sbb-checkbox-group ${sbbSpread(args)} id="sbb-checkbox-group">
     ${checkboxes(checked, disabledSingle, iconName, iconPlacement, label)}
-    ${args.required
-      ? html`<sbb-form-error slot="error">This is a required field.</sbb-form-error>`
-      : nothing}
+    ${args.required ? html`<sbb-error slot="error">This is a required field.</sbb-error>` : nothing}
```

Comment:
We seem to have cases where no form-field is present. Does to module change still make sense?

Reply:
I think yes, and probably you as well :P